pkg/operator/vtctld/deployment.go

```diff
@@ -102,6 +103,9 @@ func UpdateDeployment(obj *appsv1.Deployment, spec *Spec) {
 	// Tell Deployment to set the same labels on the Pods it creates.
 	update.Labels(&obj.Spec.Template.Labels, spec.Labels)
 
+	// Tell Deployment to set annotations on Pods that it creates.
+	update.Annotations(&obj.Spec.Template.Annotations, spec.Annotations)
```

Comment:
We need to clobber since we aren't hashing here. Since it looks like we don't already set any annotations, this can just be:

```go
obj.Spec.Template.Annotations = spec.Annotations
```

Reply:
Done.